OMCompiler/SimulationRuntime/c/gc/memory_pool.c

```diff
@@ -150,6 +150,11 @@ static void print_mem_pool(OMCMemPoolBlock* chunk) {
 
 MemPoolState omc_util_get_pool_state() {
   MemPoolState state;
+  /// If we forgot to explicitly initialize the pool, initialize it now.
```

Comment:
@mahge I'll merge this PR, but you might want to check why this is reached (only in the Makefile build, not the CMake build).